spec/helpers/util.js

```diff
@@ -60,6 +60,14 @@ function isIE() {
     return ((navigator.appName === 'Microsoft Internet Explorer') || ((navigator.appName === 'Netscape') && (new RegExp('Trident/.*rv:([0-9]{1,}[.0-9]{0,})').exec(navigator.userAgent) !== null)));
 }
 
+function getEdgeVersion() {
+    var match = /Edge\/(\d+)/.exec(navigator.userAgent);
+    if (match !== null) {
+        return new Number(match[1]);
```

Comment:
In this case its converting it to a Number object (so typeof result is "object") wasn't sure if that was intentional. The result here though will only ever be a whole number, like "12" or "13", because the `\d+` just matches `[0-9]`. If you want to convert that to a number primitive you can just do
```js
return +match[1]
```